src/components/BrowserFilter/BrowserFilter.react.js

```diff
@@ -34,7 +34,7 @@ export default class BrowserFilter extends React.Component {
     this.node = ReactDOM.findDOMNode(this);
   }
 
-  componentWillReceiveProps(props) {
+  UNSAFE_componentWillReceiveProps(props) {
```

Comment:
I'd either not change this now or use a safe method. Same thing for the other files. What do you think?

Reply:
I agree it appears when I ran a command suggested by a warning in the console by React, saying that `componentWillReceiveProps` is deprecated. I will change to the safe way

Reply:
```
backend.js:6 Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: SideEffect(NullComponent), SidebarHeader
```